testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/ejb/timer/remote/HotRodPersistentTimerServiceTestCase.java

```diff
@@ -0,0 +1,73 @@
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.as.test.clustering.cluster.ejb.timer.remote;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.TargetsContainer;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.as.test.clustering.cluster.ejb.timer.AbstractTimerServiceTestCase;
+import org.jboss.as.test.shared.CLIServerSetupTask;
+import org.jboss.shrinkwrap.api.Archive;
+import org.junit.ClassRule;
+import org.junit.rules.TestRule;
+
+/**
+ * @author Paul Ferraro
+ */
+@ServerSetup({ InfinispanServerSetupTask.class, HotRodPersistentTimerServiceTestCase.ServerSetupTask.class })
+public class HotRodPersistentTimerServiceTestCase extends AbstractTimerServiceTestCase {
+
+    @ClassRule
+    public static final TestRule INFINISPAN_SERVER_RULE = infinispanServerTestRule();
+
+    @Deployment(name = DEPLOYMENT_1, managed = false, testable = false)
+    @TargetsContainer(NODE_1)
+    public static Archive<?> deployment0() {
+        return createArchive();
+    }
+
+    @Deployment(name = DEPLOYMENT_2, managed = false, testable = false)
+    @TargetsContainer(NODE_2)
+    public static Archive<?> deployment1() {
+        return createArchive();
+    }
+
+    private static Archive<?> createArchive() {
+        return createArchive(HotRodPersistentTimerServiceTestCase.class);
+    }
+
+    static class ServerSetupTask extends CLIServerSetupTask {
+        ServerSetupTask() {
+            this.builder.node(THREE_NODES)
+                    .setup("/subsystem=infinispan/cache-container=ejb/invalidation-cache=hotrod-persistent:add")
+                    .setup("/subsystem=infinispan/cache-container=ejb/invalidation-cache=hotrod-persistent/component=expiration:add(interval=0)")
+                    .setup("/subsystem=infinispan/cache-container=ejb/invalidation-cache=hotrod-persistent/component=locking:add(isolation=REPEATABLE_READ)")
+                    .setup("/subsystem=infinispan/cache-container=ejb/invalidation-cache=hotrod-persistent/component=transaction:add(mode=BATCH)")
+                    .setup("/subsystem=infinispan/cache-container=ejb/invalidation-cache=hotrod-persistent/store=hotrod:add(remote-cache-container=ejb, cache-configuration=default, fetch-state=false, purge=false, passivation=false, shared=true)")
```

Comment:
I started an Infinispan 13.0.6 locally and this configuration didn't work until I removed `cache-configuration=default` altogether. It used to fail with the following message:

```
ISPN000374: No such template 'default' when declaring 'ROOT.war.EjbTimerBean.PERSISTENT'
```

Reply:
You are probably running against the default infinispan-server configuration.  The clustering testsuite uses a custom infinispan-server config.